docs/typescript/how-to-set-an-activity-retry-policy-in-typescript.md

```diff
@@ -0,0 +1,24 @@
+---
+id: how-to-set-an-activity-retry-policy-in-typescript
+title: How to set an Activity Retry Policy in TypeScript
+sidebar_label: Set an Activity Retry Policy
+description: Set an Activity Retry Policy
+---
+
+To set Activity Retry Policies in TypeScript, specify the `proxyActivities`, then add your configurable [Retry Policy](https://typescript.temporal.io/api/interfaces/proto.coresdk.common.iretrypolicy).
```

Comment:
Maybe more like "set the Retry Policy in an options object and then pass the object to the constructor method on a new instance of `proxyActivities`." ?